mode/common.go

```diff
@@ -13,7 +13,13 @@ import (
 )
 
 func newGRPCClient(cfg *config.Config) (grpc.Client, error) {
-	addr := fmt.Sprintf("%s:%s", cfg.Server.Host, cfg.Server.Port)
+	addr := cfg.Server.Host
+
+	// as long as the address doesn't start with unix, also add the port.
+	if !strings.HasPrefix(cfg.Server.Host, "unix://") {
```

Comment:
I would personally add --unix switch and use that to decide if we do host:port or just host

Reply:
I'm waiting on @ktr0731 to decide on the path forward.

Reply:
Poke @ktr0731 @kakke18. Any opinion on this, or reservations against this PR?